src/TestSuite/FixtureSchemaExtension.php

```diff
@@ -34,12 +35,29 @@ public function __construct()
     {
         FixtureLoader::setInstance(new FixtureDataManager());
 
-        // This isn't a great place for this, but I plan on revisiting
-        // how DB logging works in tests soon.
         $enableLogging = in_array('--debug', $_SERVER['argv'] ?? [], true);
+        $this->enableLogging($enableLogging);
         $this->aliasConnections($enableLogging);
     }
 
+    /**
+     * Enables a logger for SQL queries.
+     *
+     * @param bool $enableLogging Whether or not logging should be enabled.
+     * @return void
+     */
+    protected function enableLogging(bool $enableLogging): void
+    {
+        if (!$enableLogging) {
```

Comment:
Is the argument necessary if it's a no-op when `false`?

Reply:
I could move the conditional into constructor instead.